### PR TITLE
Update CRs formatting according to new guidelines

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Updated CRs formatting according to new guidelines - use natural language

**Related issue(s)**
See also #12473

### CODE REVIEW

File: docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ This diagram illustrates the workflow that leads to exposing a service in Kyma:
 
 ![service-exposure-flow](./assets/apix-api-gateway-flow.svg)
 
-- **API Gateway Controller** is a component responsible for exposing services. API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Controller](https://book.kubebuilder.io/) principles. API Gateway Controller listens for newly created custom resources (CR) that follow the set `apirule.gateway.kyma-project.io` custom resource definition (CRD), which describes the details of exposing services in Kyma.
+- **API Gateway Controller** is a component responsible for exposing services. API Gateway Controller is an application deployed in the `kyma-system` Namespace, implemented according to the [Kubernetes Controller](https://book.kubebuilder.io/) principles. API Gateway Controller listens for newly created custom resources (CR) that follow the set `.gateway.kyma-project.io` custom resource definition (CRD), which describes the details of exposing services in Kyma.
```
Comment (majakurcius) [inline docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md:9]:
```
(...) that follow the set `.gateway.kyma-project.io`(...)
```

Just double-checking: is this change (removal of the `apirule` prefix) intentional?

File: docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ This diagram illustrates the workflow that leads to exposing a service in Kyma:
...
 
-- **Istio Virtual Service** specifies the services visible outside the cluster. API Gateway Controller creates a Virtual Service for the hostname defined in the `apirule.gateway.kyma-project.io` CRD. The convention is to create a hostname using the name of the service as the subdomain, and the domain of the Kyma cluster. To learn more about the Istio Virtual Service concept, read this [Istio documentation](https://istio.io/latest/docs/reference/config/networking/virtual-service/).
+- **Istio Virtual Service** specifies the services visible outside the cluster. API Gateway Controller creates a Virtual Service for the hostname defined in the `.gateway.kyma-project.io` CRD. The convention is to create a hostname using the name of the service as the subdomain, and the domain of the Kyma cluster. To learn more about the Istio Virtual Service concept, read this [Istio documentation](https://istio.io/latest/docs/reference/config/networking/virtual-service/).
```
Comment (majakurcius) [inline docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md:11]:
```
(...) that follow the set `.gateway.kyma-project.io`(...)
```

(2) Just double-checking: is this change (removal of the `apirule` prefix) intentional?

File: docs/05-technical-reference/00-custom-resources/apix-01-apirule.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: APIRule
+title: API Rule
 ---
 
 The `apirule.gateway.kyma-project.io` custom resource definition (CRD) is a detailed description of the kind of data and the format the API Gateway Controller listens for. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
```
Comment (majakurcius) [inline docs/05-technical-reference/00-custom-resources/apix-01-apirule.md:5]: (3) If the change mentioned in the previous comments is intentional, should it be applied here too?

File: docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md (hunk 1 above)
Comment (mmitoraj) [reply docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md:9]: Nope, thanks for checking :)

File: docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md (hunk 2 above)
Comment (mmitoraj) [reply docs/05-technical-reference/00-architecture/apix-01-architecture-apigateway.md:11]: Nope, thanks for checking :)

**Review by majakurcius — APPROVED** [review]

**Pull request merged**